### DISCUSSION TITLE
User session count limiter - getting "We are sorry... Invalid username or password

### DISCUSSION THREAD

**Discussion by PaulGarnham-Contemi** [keycloak/keycloak#20555] [Q&A]
Need to add User session limits authentication.

I followed the steps in the Server Administration Guide on a copy of the browser built-in flow. After setting up I always get yhe 'We are sorry ..." 

I am running the Docker release (20.0.3)

I want to limit concurrent client sessions (per user) to 1 and expire older sessions.

TIA.

**Comment by andycostintoma** [comment]
I am having the same problem. Did you find a solution?

**Reply by PaulGarnham-Contemi** [reply]
Andy

Finally resolved, Updated to 23 and followed the new documentation on this.
https://www.keycloak.org/docs/latest/server_admin/#_user_session_limits

**Comment by meetsoni15** [comment]
We have same issue in production, how to solve it ?
We are using 21 version

**Comment by danielFesenmeyer** [comment]
Hi guys,

could you please provide related logs for such issues?
Otherwise it's very difficult to support.
